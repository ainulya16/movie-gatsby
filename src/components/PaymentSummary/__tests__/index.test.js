import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import PaymentComponent from '../index';

describe('Payment component', () => {
  let wrapper;
  const totalPaymentTesting = 400;
  beforeEach(() => {
    wrapper = shallow(<PaymentComponent totalPayment={totalPaymentTesting} />);
  });
  it('should be exists', () => {
    expect(wrapper).toBeDefined();
  });
  // it('should contain 1 Card component', () => {
  //   expect(wrapper.find(Card)).toHaveLength(1);
  // });
  it('should match the snapshot', () => {
    const tree = renderer.create(wrapper).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
