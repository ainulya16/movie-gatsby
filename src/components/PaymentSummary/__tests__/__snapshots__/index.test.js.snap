// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Payment component should match the snapshot 1`] = `
.c2 {
  position: relative;
  bottom: 20px;
  font-size: 22px;
  font-weight: bold;
}

.c3 {
  font-size: 45px;
  font-weight: bold;
}

.c5 {
  font-size: 22px;
  font-weight: 900;
}

.c6 {
  background-color: white;
  border: 2px solid black;
  color: black;
  padding: 5px;
  font-size: 18px;
  font-weight: 900;
}

.c7 {
  float: right;
  font-size: 20px;
  -webkit-text-decoration: line-through;
  text-decoration: line-through;
  font-weight: 900;
}

.c4 {
  margin: 15px 0px;
}

.c8 {
  font-size: 15px;
}

.c0 {
  max-width: 90%;
  background-color: #f5f5f5;
}

@media (max-width:992px) {
  .c1 {
    padding: 0 40px;
  }
}

@media (max-width:768px) {
  .c0 {
    max-width: 100%;
  }
}

<div
  className="c0 c1"
>
  
  
  <div
    className="ant-card ant-card-bordered"
  >
    <div
      className="ant-card-body"
      style={Object {}}
    >
      <div
        className="ant-row"
      >
        <div
          className="ant-col ant-col-8"
        >
          <span
            style={
              Object {
                "fontSize": "20px",
                "fontWeight": "bold",
              }
            }
          >
            Total
          </span>
          <h6
            style={
              Object {
                "fontSize": "25px",
                "textTransform": "capitalize",
              }
            }
          >
            Payment
          </h6>
        </div>
        <div
          className="ant-col ant-col-8 ant-col-offset-8"
        >
          <div
            style={
              Object {
                "float": "right",
              }
            }
          >
            <span
              className="c2"
            >
              $
            </span>
            <span
              className="c3"
            >
              400
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="ant-row c4"
  >
    <div
      className="ant-col ant-col-16"
    >
      <span
        className="c5"
      >
        Coupon Code
      </span>
      <span
        className="c6"
        style={
          Object {
            "marginLeft": "10px",
          }
        }
      >
        15OFF
      </span>
    </div>
    <div
      className="ant-col ant-col-8"
    >
      <span
        className="c7"
      >
        $550
      </span>
    </div>
  </div>
  <p
    className="c8"
  >
    Enim esse exercitation adipisicing ut ex Lorem dolor labore consequat incididunt. Elit nulla amet ea dolore elit magna quis ullamco proident eiusmod cillum velit in et. Ad laborum consequat velit sunt duis commodo cillum. Est mollit consequat pariatur amet culpa qui commodo cupidatat autre. Ad elit cillum id in nostrud consectetur aliqua anim ad reprehenderit ullamco minim.
  </p>
</div>
`;
